doctr/utils/geometry.py

```diff
@@ -44,3 +45,13 @@ def resolve_enclosing_bbox(bboxes: List[BoundingBox]) -> BoundingBox:
 def resolve_enclosing_rbbox(rbboxes: List[RotatedBbox]) -> RotatedBbox:
     pts = np.asarray([pt for rbbox in rbboxes for pt in rbbox_to_polygon(rbbox)], np.float32)
     return fit_rbbox(pts)
+
+
+def resolve_enclosing_bboxarray(bboxarray: np.ndarray) -> np.ndarray:
```

Comment:
I suggest naming it `enclose_boxes` :) 

I was wondering how it differs from `resolve_enclosing_bbox`

Reply:
it has the same purpose but the computation is from an array of boxes instead of a list of `BoundingBox`